hw/dv/tools/testplans/csr_testplan.hjson

```diff
@@ -64,7 +64,21 @@
             - Shuffle the list of CSRs first to remove the effect of ordering.
             '''
       milestone: V1
-      tests: ["{name}{intf}_csr_aliasing",]
+      tests: ["{name}{intf}_csr_aliasing"]
+    }
+    {
+      name: csr_mem_rw_with_rand_reset
+      desc: '''
+            Verify random reset during CSR/memory access.
+            - Run csr_rw sequence to randomly access CSRs
+            - If memory exists, run mem_partial_access in parallel with csr_rw
```

Comment:
Does the test actually prove the mem was also reset properly?

Reply:
I updated cip_base_vseq to use `clone` to preserve mem data from last run, also make sure only save write data when reset isn't active. Now it will verify mem is reset properly.